components/micropython/micropython.mk

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2025, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# This Makefile snippet builds the MicroPython component
+#
+# NOTES:
+# Generates micropython.elf
+# Requires variables:
+#	MICROPYTHON_LIBMATH
+#	MICROPYTHON_CONFIG_INCLUDE
+# Optional variables:
+#	MICROPYTHON_FROZEN_MANIFEST
+#	MICROPYTHON_EXEC_MODULE
+#	MICROPYTHON_ENABLE_I2C
+#	MICROPYTHON_ENABLE_FRAMEBUFFER
+#	MICROPYTHON_ENABLE_VFS_STDIO
+#	MICROPYTHON_ENABLE_SERIAL_STDIO
+
+MICROPYTHON_DIR := $(realpath $(dir $(lastword $(MAKEFILE_LIST))))
+
+micropython.elf: FORCE mpy-cross ${LIONSOS}/dep/libmicrokitco/Makefile $(MICROPYTHON_FROZEN_MANIFEST) $(MICROPYTHON_EXEC_MODULE)
+	make -C $(MICROPYTHON_DIR) \
```

Comment:
use `${MAKE}` not `make`
Because 'FORCE' is on it'll always be executed, so you need no other dependencies, unless you want to make sure they're built before executing this line.